advanced/Exercise_8.md

```diff
@@ -1,59 +1,69 @@
-# Exercise 8 - Using git subtree
+# Exercise 8 - Using git submodule
 
-This exercise is designed to be paired with Exercise 7 to help you compare and contrast the two main methods for embedding a git repository into another git repository. In this exercise, we will learn to use `git subtree` to nest a git repository inside another one. We will first add the git subtree and examine how the host and nested repositories deal with changes in their respective files. We will then learn how to pull and push changes to and from the sub-repository.
+This exercise is designed to be paired with Exercise 8 to help you compare and contrast the two main methods for embedding a git repository into another git repository. This exercise illustrates the submodule method, while the next exercise illustrates the subtree method. In this exercise, we will learn to use git submodule to nest a git repository inside another one. We will first add the git submodule and examine how the host and nested repositories deal with changes in their respective files. We will then learn how to pull and push changes to and from the sub-repository.
```

Comment:
to be paired with Exercise 8?

Reply:
Removed in [2104b09](https://github.com/C2SM/git-course/pull/83/commits/2104b09d22dc5393175e7d12d63de02d12ea18f0)